### DISCUSSION TITLE
JReleaser package with Installer

### DISCUSSION THREAD

**Discussion by ebresie** [jreleaser/jreleaser#888] [General]
Maybe a new user question…

With the ability for a release to create multiple Target packages in different format, as part of this, does it allows inclusion of an target specific installer as well (i.e. Windows installer like Inno Setup, etc.), Linux installer, etc.) which allows setup for each target environment?  

Or is this out of scope of this project?

**Comment by aalmiray** [comment]
It depends. If the installer is packaged with any of the file formats supported by the `NATIVE_PACKAGE` distribution https://jreleaser.org/guide/latest/distributions/native-package.html then yes, you can release the installer with additional packager support (https://jreleaser.org/guide/latest/integrations.html#_package_publish).

If the file format is not supported _and_ you don't need additional packager support _then_ you can configure the file using the [files](https://jreleaser.org/guide/latest/configuration/files.html) section, instead of [distributions](https://jreleaser.org/guide/latest/configuration/distributions.html).
